frontend/Auth/Login.tsx

```diff
@@ -1,9 +1,165 @@
+  >(
+    (evt) => {
+      evt.preventDefault();
+      let requestPromise = flaxFetch<LoginResultData>(`/login`, {
+        method: "POST",
+        body: {
+          username: loginFormData.username,
+          password: loginFormData.password,
+        },
+      });
+      return requestPromise;
+    },
+    {
+      onSuccess: async (data, variables, context) => {
+        history.push("/home");
+      },
+    }
+  );
 
-export function Login(props) {
   return (
-    <Form method="post" action="/login">
-      <Button type="submit">Login</Button>
-    </Form>
+    <div style={{ display: "flex", justifyContent: "center" }}>
+      <Card
+        style={{
+          padding: "10px",
+          width: "30%",
+          boxShadow: "none",
+          marginTop: "15%",
+        }}
+      >
+        <h1 style={{ color: "#646461", fontSize: "20px" }}>Sign in</h1>
+        <Form onSubmit={unary(submitMutation.mutate)}>
+          <Form.Field>
+            <label htmlFor="username">Email</label>
+            <input
+              id="username"
+              value={loginFormData.username}
+              onChange={(evt) =>
+                setLoginFormData({
+                  ...loginFormData,
+                  username: evt.target.value,
+                })
+              }
+              required
+            />
+          </Form.Field>
+
+          <Form.Field>
+            <label htmlFor="password">Password</label>
+            <input
+              id="password"
+              value={loginFormData.password}
+              onChange={(evt) =>
+                setLoginFormData({
+                  ...loginFormData,
+                  password: evt.target.value,
+                })
+              }
+              required
+            />
+          </Form.Field>
+
+          <Button
+            type="submit"
+            style={{
+              background: "#2a467b",
+              color: "white",
+              width: "50%",
+              margin: 0,
+            }}
+          >
+            Sign in
+          </Button>
+
+          <Button
+            type="submit"
+            style={{
+              background: "white",
+              color: "#2a467b",
+              width: "50%",
+              margin: 0,
+            }}
+          >
+            Reset Password
+          </Button>
+
+          <div style={{ marginTop: "20px", marginBottom: "10px" }}>
+            <div
+              style={{
+                width: "100%",
+                height: "10px",
+                textAlign: "center",
+                borderBottom: "1px solid #b3b3b1",
+                color: "#b3b3b1",
+              }}
+            >
+              <span
+                style={{
+                  fontSize: "10px",
+                  backgroundColor: "white",
+                  padding: "0 10px",
+                }}
+              >
+                or
+              </span>
+            </div>
+          </div>
+
+          <Button
```

Comment:
This should be `<Button type="button">`. The default value for the `type` attribute is `"submit"` and we don't want clicking on the sign in with google button to call the onSubmit handler.